db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -581,7 +586,7 @@ void DBImpl::NotifyOnFlushBegin(ColumnFamilyData* cfd, FileMetaData* file_meta,
     info.triggered_writes_slowdown = triggered_writes_slowdown;
     info.triggered_writes_stop = triggered_writes_stop;
     info.smallest_seqno = file_meta->fd.smallest_seqno;
-    info.largest_seqno = file_meta->fd.largest_seqno;
+    info.largest_seqno = largest_seqno;
```

Comment:
Why `file_meta->fd.largest_seqno` doesn't work?

Reply:
It was initialized as 0 at the beginning of flush job.